apps/meteor/app/e2e/client/rocketchat.e2e.ts

```diff
@@ -264,8 +264,9 @@ class E2E extends Emitter {
 			return null;
 		}
 
-		if (!this.instancesByRoomId[rid]) {
-			this.instancesByRoomId[rid] = new E2ERoom(Meteor.userId(), room);
+		const userId = Meteor.userId();
```

Comment:
no other way besides using Meteor to get this userId?

Reply:
Actually,not sure is needed. But didn't want to touch anything besides the typings here.